packages/labs/src/components/timezone-picker/timezonePicker.tsx

```diff
@@ -60,6 +60,12 @@ export interface ITimezonePickerProps extends IProps {
      */
     showLocalTimezone?: boolean;
 
+    /**
+     * Whether to use manual calculation (faster, but possibly less accurate) rather than moment-timezone to infer metadata;
```

Comment:
- [ ] typo: Add trailing period, not semicolon.
- [ ] Also, can you add language clarifying what is less accurate when `useManualCalc=true`?